test/test-kitchen.ts

```diff
@@ -113,6 +113,10 @@ test.serial('use as a non-locally installed module', async t => {
   // server.ts has a lint error. Should error.
   await t.throws(simpleExecp(`${GTS} check src/server.ts`, opts));
 
+  // Compilation shouldn't have happened. Hence no `build` directory.
+  const dirContents = fs.readdirSync(`${tmpDir.name}/kitchen`);
+  t.is(dirContents.indexOf('build'), -1);
```

Comment:
Is it possible that `npm run compile` was run but had an error that caused the `build` directory to not be created?

Reply:
I verified that the `build` directory is created without my change.